parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -331,6 +358,14 @@ write_files:
             endscript
     }
 
+- path: /etc/cron.d/logrotate
```

Comment:
nit: I personally prefer systemd timers.

also isn't there an existing cronjob? is this overriding that?

Reply:
Yep, this is one commonly used method of overriding the daily cron. I'll look into systemd timers.